google/cloud/aiplatform/experimental/vertex_model/base.py

```diff
@@ -160,10 +147,22 @@ def f(*args, **kwargs):
             valid_types = [int, float, str] + list(
                 obj._data_serialization_mapping.keys()
             )
+
             if parameter_type not in valid_types:
-                raise RuntimeError(
-                    f"{parameter_type} not supported. parameter_name = {parameter_name}. The only supported types are {valid_types}"
-                )
+                if parameter_type.__name__ == "DataLoader":
+                    obj._data_serialization_mapping[parameter_type] = (
+                        pytorch._deserialize_dataloader,
```

Comment:
Using these modules here will cause import exceptions when Pandas and Torch are not installed. One possible solution: Create an `__init__` file in serializers with a build map method and require each library specific module keep a map constant in the module.

Something like:

`.../serializers/pandas.py`
```Python
...
_DATA_SERIALIZER_MAP = {pd.Dataframe: (pandas._deserialize_dataframe, pandas._serialize_dataframe)}
```

`.../serializers/__init__.py`
```Python
import sys
from google.cloud import aiplatform

module_names = ['pandas', 'pytorch']

def build_map_safe():
  serializer_map = {}
  for module_name in module_names:
    try:
      module = sys.modules.get(f'google.cloud.aiplatform.experimental.vertex_model.serializers.{module_name}') 
      serializer_map.update(module._DATA_SERIALIZER_MAP)
    except ImportError:
      pass 

  return serializer_map
```

Reply:
Can discuss during meeting, but a little confused as to what next steps would be to maintain the mapping after initializing this __init__ file.